README.md

```diff
@@ -2,7 +2,37 @@
 
 Web3 ProviderEngine is a tool for composing your own [web3 providers](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3).
 
-<table><tr><td><p align="center"><b>⚠️ PLEASE READ ⚠️</b></p><div align="center">This package was originally created for MetaMask and is being phased out in favor of <a href="https://www.npmjs.com/package/@metamask/json-rpc-engine"><code>@metamask/json-rpc-engine</code></a> and <a href="https://www.npmjs.com/package/@metamask/eth-json-rpc-middleware"><code>@metamask/eth-json-rpc-middleware</code></a>. As such, we will no longer be accepting changes to this package except those which address security issues.</div></td></tr></table>
+> [!CAUTION]
+> This package has been deprecated.
+>
+> This package was originally created for MetaMask, but has been replaced by `@metamask/json-rpc-engine`, `@metamask/eth-json-rpc-middleware`, and `@metamask/eth-json-rpc-provider`.
+>
+>Here is an example of how to create a provider using those packages:
+>
+> ```js
+>import { providerFromMiddleware } from '@metamask/eth-json-rpc-provider';
+>import { createFetchMiddleware } from '@metamask/eth-json-rpc-middleware';
+>
+>const rpcUrl = '[insert RPC URL here]';
+>
+>const fetchMiddleware = createFetchMiddleware({ rpcUrl });
```

Comment:
This is a bit incomplete, but the following should work (I tested this in CodeSandbox):

~~~ suggestion
>import { providerFromMiddleware } from '@metamask/eth-json-rpc-provider';
>import { createFetchMiddleware } from '@metamask/eth-json-rpc-middleware';
>import { valueToBytes, bytesToBase64 } from '@metamask/utils';
>import fetch from 'cross-fetch';
>
>const rpcUrl = '[insert RPC URL here]';
>
>const fetchMiddleware = createFetchMiddleware({
>  btoa: (stringToEncode) => bytesToBase64(valueToBytes(stringToEncode)),
>  fetch,
>  rpcUrl,
>});
~~~

Reply:
Could you elaborate on what this is meant to do? Is this an attempt to get this snippet working for both web and node environments?

I did confirm that my original snippet works in an envirionment with a `btoa` and `fetch` global

Reply:
Yes, that was the motivation, for this to work cross-browser. Or at least in the browser, since the `btoa` global seems restricted to web workers (from what I read).

If you've tested this though then I'm okay with it.

Reply:
Nah that makes sense, I've updated the example to use your suggested changes: https://github.com/MetaMask/web3-provider-engine/pull/469/commits/b9492e7ce03d13b76ca685abdcc29c6a0b27ea38